packages/cardano-services/test/Program/services/ogmios.test.ts

```diff
@@ -28,6 +28,21 @@ import { types } from 'util';
 import axios from 'axios';
 import http from 'http';
 
+export const getAliceHandleProviderResponse: HandleResolution = {
```

Comment:
This is unavailable because it's in a utils file for a test folder. I think it should be moved, as utils for the KoralabsHandleProvider? or somewhere else where it's available.

Reply:
i.e this is a duplicate because the other mock object isn't able to be imported? Did you try adding a path to the tsconfig?

Reply:
I did try. But it wont' work, because the `getAliceHandleProviderResponse` gets exported from a `test/utils` file. So in order to get the import to work, the path should be something like: `@cardano-sdk/cardano-services-client/test/utils` and I believe this is something we want to avoid. 
Either the 'utils' file should be moved in a 'utils' folder with an index file. And even so, I'm not sure we export utils from tests to other folders, I think they are just for testing purposes. 
Or we then move it to another exported folder altogether.

Reply:
ebc0237c005b9b050f038dcac487ca1e9a5875e6 should solve  this